en/project-management/project-selection.rst

```diff
@@ -0,0 +1,40 @@
+Project selection
+=================
+
+QField opens QGIS projects in the ".qgs" format. Currently it is not possible to open ".qgz" files.
+
+OField uses a customized file selector to open the projects. On choosing "Open project" from the QField menu, the file selector will be opened showing the roots:
```

Comment:
"showing the following folders:" ?

I think root as a term is not widely known (except for (arch) linux geniouses)

Reply:
:)

Reply:
also root is understood as rooting

Reply:
yes, already changed.